src/__tests__/fireEvent.test.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {
+  View,
+  TouchableOpacity,
+  Text,
+  ScrollView,
+  TextInput,
+} from 'react-native'; // eslint-disable-line import/no-unresolved
+import fireEvent from '../fireEvent';
+import { render } from '..';
```

Comment:
`fireEvent` is exported from `index.js` so we should import from there, which will also serve as API test